### PR TITLE
Give some time indication when doing mass-branching

### PR DESCRIPTION
**Pull request by pypingou**

### CODE REVIEW

**Comment by puiterwijk** [comment]
Perhaps also print the name of a package when it has been branched to see progress?

File: utility/pkgdb2_branch.py [hunk 1]
```diff
@@ -185,6 +188,9 @@ def main():
     except pkgdb2.lib.PkgdbException, err:
         print err
         return 1
+    end = datetime.datetime.utcnow()
+    print "Finished at %s" % end
+    print "Last for %s" % (end - start)
```
Comment (puiterwijk) [inline utility/pkgdb2_branch.py]: s/Last/Lasted/

**Comment by pypingou** [comment]
> Perhaps also print the name of a package when it has been branched to see progress?

We can't do that anymore as the entire branching is now done via a two SQL queries which make the process much faster than iterating through the DB with python.

**Comment by puiterwijk** [comment]
Ah, right. Fair enough. :+1:

**Comment by puiterwijk** [comment]
:+1:

**Comment by pypingou** [comment]
Thanks! :)

**Pull request merged**